proxy-node-gateway/src/main/java/uk/gov/ida/notification/session/GatewaySessionData.java

```diff
@@ -30,6 +38,26 @@ public GatewaySessionData(
         this.eidasDestination = eidasSamlParserResponse.getDestination();
         this.eidasRelayState = eidasRelayState;
         this.eidasConnectorPublicKey = eidasSamlParserResponse.getConnectorEncryptionPublicCertificate();
```

Comment:
Presume that Redis can persist certs as Strings without mangling them?  Doesn't need to be base64?

Reply:
Me and @dbes-gds have decided to move this into a new ticket.

Reply:
Has this ticket been created?